src/freenet/clients/fcp/ClientGet.java

```diff
@@ -925,14 +924,17 @@ public boolean filterData() {
 	synchronized RequestStatus getStatus() {
 		boolean totalFinalized = false;
 		int total = 0, min = 0, fetched = 0, fatal = 0, failed = 0;
+		Date latestSuccess = CurrentTimeUTC.get(), latestFailure = null;
```

Comment:
Multiple variables in a single declaration is [not allowed by the coding standards](https://google-styleguide.googlecode.com/svn/trunk/javaguide.html#s4.8.2.1-variables-per-declaration).